Android/GoogleDagger/source/Dagger/Dagger.csproj

```diff
@@ -41,4 +42,12 @@
     <Reference Include="Mono.Android" />
   </ItemGroup>
 
+  <ItemGroup>
+    <Compile Update="Classes.cs">
+      <SubType></SubType>
+    </Compile>
+  </ItemGroup>
```

Comment:
Maybe remove this chunk

Reply:
Done. Also changed file name to `Additions` to match other places